### PR TITLE
[GH-149] Send ephemeral message after /jira connect flow succeeds

### PR DESCRIPTION
**Pull request by promulo**
#### Summary
This PR sends an ephemeral message to the user after the `/jira connect` flow is completed. Please check [this comment](https://github.com/mattermost/mattermost-plugin-jira/issues/149#issuecomment-716924011) regarding this issue, as this PR does not address all the requirements.

#### Ticket Link
GH-149

### CODE REVIEW

**Review by aaronrothschild — CHANGES_REQUESTED** [review]
Thanks for the contribution @promulo !

**Comment by jasonblais** [comment]
@aaronrothschild did you mean to accept or request changes?

**Comment by jasonblais** [comment]
cc @levb @jfrerich for dev review

**Comment by aaronrothschild** [comment]
> @aaronrothschild did you mean to accept or request changes?

@promulo Sorry, I mean to request the small wording change for the user. Otherwise, all good.

**Comment by promulo** [comment]
Sorry @aaronrothschild but what would be the wording change here? Do you mean the inclusion of the user e-mail in the message?

File: webapp/src/actions/index.ts [hunk 1]
```diff
@@ -424,6 +424,10 @@ export function handleConnectChange(store) {
             return;
         }
 
+        if (msg.event === 'custom_jira_connect') {
+            store.dispatch(sendEphemeralPost('You have successfully connected your Jira account.'));
```
Comment (aaronrothschild) [inline webapp/src/actions/index.ts:428]:
```suggestion
            store.dispatch(sendEphemeralPost('You have successfully connected your Jira account. Type in /jira to get started. '));
```

Comment (aaronrothschild) [reply webapp/src/actions/index.ts:428]: Sorry - Github gave me some issues today.... the comment didn't post.  Added it now. small change to text of what bot says.

**Review by aaronrothschild — APPROVED** [review]
Awesome, thanks!

**Review by levb — APPROVED** [review]

**Review by jfrerich — APPROVED** [review]

**Comment by DHaussermann** [comment]
Thanks @promulo  for this PR. Sorry for the delay in testing. 

This is certainly an improvement in the UX. One Minor point is that it would feel more consistent if the ephemeral post was made by the Jira bot rather than a system post. I don't know if this is a minor change or something we should make a separate enhancement issue for?   

Screen Shot 2020-11-19 at 9 29 44 AM

@aaronrothschild Should I create a separate issue for this?

**Comment by promulo** [comment]
Hi @DHaussermann, just as FYI there is already an issue for that: https://github.com/mattermost/mattermost-plugin-jira/issues/560 :)

**Comment by DHaussermann** [comment]
My mistake thanks @promulo for pointing that out 😂

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Ephemeral post confirms connection as expected
- Tested when connecting to both Jira Cloud and Server instaces
- https://github.com/mattermost/mattermost-plugin-jira/issues/560 will address posts not being made by the Jira bot.

LGTM!

**Comment by hanzei** [comment]
Thanks @promulo :+1:

**Pull request merged**